Mathlib/Algebra/Group/Action/Defs.lean

```diff
@@ -71,14 +71,34 @@ lemma op_smul_eq_mul {α : Type*} [Mul α] (a b : α) : MulOpposite.op a • b =
 @[to_additive (attr := simp)]
 lemma MulOpposite.smul_eq_mul_unop [Mul α] (a : αᵐᵒᵖ) (b : α) : a • b = b * a.unop := rfl
 
-/-- Type class for additive monoid actions. -/
+/--
+Type class for additive monoid actions on types, with notation `g +ᵥ p`.
+
+The `AddAction G P` typeclass says that the additive monoid `G` acts additively on a type `P`.
+More precisely this means that the action satisfies the two axioms `0 +ᵥ p = p` and
+`(g₁ + g₂) +ᵥ p = g₁ +ᵥ (g₂ +ᵥ p)`. A mathematician might simply say that the monoid `G`
+acts on `P`.
+
+For example, if `A` is an additive group and `X` is a type, if a mathematician says
+say "let `A` act on the set `X`" they will usually mean `[AddAction A X]`.
+-/
 class AddAction (G : Type*) (P : Type*) [AddMonoid G] extends VAdd G P where
   /-- Zero is a neutral element for `+ᵥ` -/
   protected zero_vadd : ∀ p : P, (0 : G) +ᵥ p = p
   /-- Associativity of `+` and `+ᵥ` -/
   add_vadd : ∀ (g₁ g₂ : G) (p : P), (g₁ + g₂) +ᵥ p = g₁ +ᵥ g₂ +ᵥ p
 
-/-- Typeclass for multiplicative actions by monoids. This generalizes group actions. -/
+/--
+Type class for multiplicative monoid actions on types, with notation `g • p`.
+
+The `MulAction G P` typeclass says that the monoid `G` acts multiplicatively on a type `P`.
```

Comment:
I'm ok with dropping "multiplicative" here, with the premise the mathlib assumes things are multiplicative by default.